src/app.ts

```diff
@@ -13,14 +13,62 @@ interface TemperatureSummary {
   average: number
 }
 
+const readingsFormatted: Map<string, Map<string, TemperatureReading[]>> =
+  new Map()
+
+// Store temperature readings in a sorted way to facilitate searches by city and time
 export function processReadings(readings: TemperatureReading[]): void {
-  // add here your code
+  readings.forEach((r) => {
+    const { city, time } = r
+
+    const timeF = new Intl.DateTimeFormat('en-US').format(time)
```

Comment:
I'm taking a guess, but does timeF stands for formatted Time?
If that's the case I'd suggest calling it that same way `formattedTime`. It's a nit, and doesn't make any difference in terms of code, but helps a lot to understand faster what we are doing